x-pack/plugins/fleet/cypress/integration/fleet_startup.spec.ts

```diff
@@ -13,7 +13,7 @@ describe('Fleet startup', () => {
     navigateTo(FLEET);
   });
 
-  it('should display Add agent button and Healthy agent once Fleet Agent page loaded', () => {
+  it('should display Add agent button and Healthy agent once Fleet Agent page loaded #2', () => {
```

Comment:
why the change here?

Reply:
It was a typo, thanks for spotting it. Thanks!